src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -996,23 +996,44 @@ private SymbolAndDiagnostics ComputeImplementationAndDiagnosticsForInterfaceMemb
             {
                 if ((object)implicitImpl != null)
                 {
-                    if (!canBeImplementedImplicitlyInCSharp9)
+                    bool suppressRegularValidation = false;
+
+                    if (!canBeImplementedImplicitlyInCSharp9 && interfaceMember.Kind == SymbolKind.Method &&
+                        (object)implementingBaseOpt == null)  // Otherwise any approprite errors are going to be reported for the base.
```

Comment:
```suggestion
                        (object)implementingBaseOpt == null)  // Otherwise any appropriate errors are going to be reported for the base.
```